mime_type.rb

```diff
@@ -17,16 +17,19 @@ def init_associations
     associations
   end
 
+  def match_file(filename)
+    splitted_fname = filename.split('.')
+    result = 'UNKNOWN'
+    if splitted_fname.size > 1 && filename[-1] != '.'
+      result = associations.fetch(splitted_fname.last.downcase, 'UNKNOWN')
```

Comment:
You can simplify the logic here:
You start with a result of `'UNKNOWN'`. You check some condition and if it's true, you use Hash#fetch to either return a result, or provide the same default value of 'UNKNOWN'.

Try refactoring it in a way that would use 'UNKNOWN' only once